src/libxtp/gwbse/gw.cc

```diff
@@ -328,18 +324,21 @@ double GW::SolveQP_Bisection(double lowerbound, double f_lowerbound,
   while (true) {
     double c = 0.5 * (lowerbound + upperbound);
     if (std::abs(upperbound - lowerbound) < _opt.g_sc_limit) {
-      break;
       zero = c;
+      break;
     }
     std::pair<double, double> temp = f(c);
     double y_c = temp.first;
     if (std::abs(y_c) < _opt.g_sc_limit) {
-      break;
       zero = c;
+      break;
```

Comment:
If this breaks the while-loop, in most cases, it would've broken at the next iteration anyways. So this break might be slightly inefficient?

Reply:
it is slightly more efficient, I just translated the wikipedia pseudo code